packages/client/src/v2-events/components/forms/inputs/DocumentViewer.stories.tsx

```diff
@@ -44,7 +45,7 @@ export const DocumentViewerSingleOption: StoryObj<typeof DocumentViewer> = {
       {
         value: {
           filename: 'fish.svg',
-          url: getUnsignedFileUrl('fish.svg'),
+          url: getUnsignedFileUrl('/ocrvs/fish.svg' as FullDocumentPath),
```

Comment:
All file references should always be in the `/:bucket/:file-identifier` format. Never in plain `:file-identifier` as the client really shouldn't need to worry about storage buckets